client/main.lua

```diff
@@ -395,72 +442,76 @@ RegisterNetEvent('garbagejob:client:SetWaypointHome', function()
 end)
 
 RegisterNetEvent('qb-garbagejob:client:RequestRoute', function()
+    if garbageVehicle then continueworking = true TriggerServerEvent('garbagejob:server:PayShift', continueworking) end
     QBCore.Functions.TriggerCallback('garbagejob:server:NewShift', function(shouldContinue, firstStop, totalBags)
         if shouldContinue then
-
-            local coords = Config.Locations["vehicle"].coords
-            QBCore.Functions.TriggerCallback('QBCore:Server:SpawnVehicle', function(netId)
-                local veh = NetToVeh(netId)
-                TaskWarpPedIntoVehicle(PlayerPedId(), veh, -1) -- hopefully this fixes an issue if something is delayed they'll get crushed
-                SetVehicleEngineOn(veh, true, true)
-                garbageVehicle = veh
-                SetVehicleNumberPlateText(veh, "GARB" .. tostring(math.random(1000, 9999)))
-                SetEntityHeading(veh, coords.w)
-                exports['LegacyFuel']:SetFuel(veh, 100.0)
-                SetVehicleFixed(veh)
-                SetEntityAsMissionEntity(veh, true, true)
-                TriggerEvent("vehiclekeys:client:SetOwner", QBCore.Functions.GetPlate(veh))
-                currentStop = firstStop
-                currentStopNum = 1
-                amountOfBags = totalBags
-                SetGarbageRoute()
-                QBCore.Functions.Notify(Lang:t("info.deposit_paid", { value = Config.TruckPrice }))
-                QBCore.Functions.Notify(Lang:t("info.started"))
-            end, Config.Vehicle, coords, true)
+            if not garbageVehicle then
+                local occupied = false
+                local coords = Config.Locations["vehicle"].coords
+                for _,v in pairs(Config.Locations["vehicle"].coords) do
+                    if not IsAnyVehicleNearPoint(vector3(v.x,v.y,v.z), 2.5) then
+                        QBCore.Functions.TriggerCallback('QBCore:Server:SpawnVehicle', function(netId)
+                            local veh = NetToVeh(netId)
+                            SetVehicleEngineOn(veh, false, true)
+                            garbageVehicle = veh
+                            SetVehicleNumberPlateText(veh, "QB-" .. tostring(math.random(1000, 9999)))
+                            SetEntityHeading(veh, coords.w)
```

Comment:
```suggestion
                            SetEntityHeading(veh, v.w)
```
Pretty sure it should be this?